product-plane-services/policy-api/src/main/java/org/opendatamesh/platform/pp/policy/api/controllers/AbstractPolicyEvaluationResultController.java

```diff
@@ -273,8 +272,8 @@ public abstract PolicyEvaluationResultResource createPolicyEvaluationResult(
             @ApiResponse(
                     responseCode = "404",
                     description = "[Not Found](https://www.rfc-editor.org/rfc/rfc9110.html#name-404-not-found)"
-                            + "\r\n - Error Code 40401 - PolicyEvaluationResult not found"
-                            + "\r\n - Error Code 40401 - Parent Policy not found",
+                            + "\r\n - Error Code 40402 - Parent Policy not found"
```

Comment:
Maybe the description/response code could be retrieved by the Enum itself, so you have only one point of configuration.